pingcap/tiflow
# checker(dm): add downstream CDC/PiTR check for lightning

Signed-off-by: lance6716 <lance6716@gmail.com>



### What problem does this PR solve?


Issue Number: ref #3510

### What is changed and how it works?

Also I remove a local disk checking in this PR, because precheck is performed at DM-master and the lightning is run at DM-worker.

### Check List 

#### Tests 

 - Manual test (add detailed scripts or steps below)
 ```
                        {
                                "id": 28,
                                "name": "lightning_downstream_cdc_pitr",
                                "desc": "check whether the downstream has tasks incompatible with Physical import mode",
                                "state": "fail",
                                "errors": [
                                        {
                                                "severity": "fail",
                                                "short_error": "found CDC capture(s): clusterID: default captureID(s): [12e1ad9b-b461-48ba-83d8-9dc4d226001b b5def693-5849-4a54-974a-52fa34869273],\nlocal backend is not compatible with them. Please switch to tidb backend then try again."
                                        }
                                ],
                                "instruction": "you can switch to logical import mode which has no requirements on this"
                        }
```

#### Questions 

##### Will it cause performance regression or break compatibility?

##### Do you need to update user documentation, design documentation or monitoring documentation?

### Release note 

```release-note
 `None`.
```
